logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -524,11 +566,12 @@ sealed class Event(open val id: String, open val transient: Boolean) {
             val userId: UserId,
             val timestampIso: String = DateTimeUtil.currentIsoDateTimeString() // TODO we are not receiving it from API
         ) : User(id, transient) {
-            override fun toString(): String {
+            override fun toLogString(): String {
                 val properties = mapOf(
-                    "id" to id.obfuscateId(),
-                    "userId" to "${userId.value.obfuscateId()}@${userId.domain.obfuscateDomain()}",
-                    "timestampIso" to timestampIso
+                    typeKey to this::class.qualifiedName,
+                    idKey to id.obfuscateId(),
```

Comment:
```suggestion
                    idKey to id.toLogString(),
```

Reply:
`id` is just `String` here. That wouldn't work

Reply:
One potential benefit about the introduction of `toLogString` is that we could add a parameter `fun toLogString(safeLoggingEnabled: Boolean = Logging.safeLoggingEnabled)`. which would allow us to turn off obfuscation when running builds locally.

This only works though if we always use `toLogString()` consistently at all levels.

Reply:
Mm, I guess it's fine if the id is one off.

If we ever want do the "safeLoggingEnabled" feature we could introduce a value class in cases like this.